inventory.js

```diff
@@ -87,24 +108,48 @@ function loadFoodItems(queryParam) {
     // Loop over the food items and add one row per item.
     $.each(foods, (i) => {
       // Add one row per item to the hidden 'allitems' container.
-      o = $('itemheader itemrow').clone().appendTo($('allitems'));
+      const o = $('itemheader itemrow').clone().appendTo($('allitems'));
+
       // Set the category id as an attribute.
       o.attr('category_id', foods[i].category_id);
 
       // Format the name and info. We reformat some all uppercase parts and replace acronyms with their full words.
       let { name } = foods[i];
       let subInfo = '';
+      // Prices may change, if the item is a particular spice or tea
+      let memberPrice = foods[i].member_price;
+      let nonMemberPrice = foods[i].nonmember_price;
+
       o.attr('name', name.toUpperCase());
 
       if (foods[i].name.indexOf('/') > -1) {
         name = foods[i].name.substr(0, foods[i].name.indexOf('/'));
         subInfo = foods[i].name.substr(foods[i].name.indexOf('/') + 1);
       }
 
+      // Spices and teas should be priced by ounces, rather than pounds.
+      // First, we'll use these regexes to check if the prices need to be adjusted -
+      // all will begin with either 'Spices,' or 'Tea,' and will include '(BY_POUND) '.
+      // Note the trailing space following '(BY_POUND) ' - at the moment, all of the names
+      // that include a weight designation have that trailing space... hence the use of .trim().
+      // Other categories, such as 'Produce (by Pound)' should also be lowercased.
+      // @author darren
+      if (name.indexOf('BY POUND') > -1 && (spiceRegex).test(name.trim()) || (teaRegex).test(name.trim())) {
+        name = name.replace('BY POUND', 'by ounce');
+        memberPrice = convertPriceUnit(foods[i].member_price, poundToOunceDenominator);
+        nonMemberPrice = convertPriceUnit(foods[i].nonmember_price, poundToOunceDenominator);
+      } else if (name.indexOf('BY POUND') > -1) {
```

Comment:
Can we move this in to the php code where we do all the other replacements? I'd like to keep that in one place. Also wondering if the conversion to ounce could be done in the php part.